metpy/calc/indices.py

```diff
@@ -101,3 +101,132 @@ def mean_pressure_weighted(pressure, *args, **kwargs):
+        V component of the wind
+    heights : array-like
+        Heights from sounding
+
+    Returns
+    -------
+    RM_vector
+        U and v component of Bunkers RM storm motion
+    LM_vector
+        U and v component of Bunkers LM storm motion
+    mean_vector
+        U and v component of sfc-6km mean flow
+
+    """
+    ums = u.to('m/s')
+    vms = v.to('m/s')
+
+    # mean wind from sfc-6km
+    u6m, v6m = mean_pressure_weighted(pressure, ums, vms, heights=heights,
+                                      depth=6000 * units('meter'))
+
+    # mean wind from sfc-500m
+    u5m, v5m = mean_pressure_weighted(pressure, ums, vms, heights=heights,
+                                      depth=500 * units('meter'))
+
+    # mean wind from 5.5-6km
+    u55m, v55m = mean_pressure_weighted(pressure, ums, vms, heights=heights,
+                                        depth=500 * units('meter'),
+                                        bottom=heights[0] + 5500 * units('meter'))
+
+    # Calculate the shear vector from sfc-500m to 5.5-6km
+    u6shr = u55m - u5m
+    v6shr = v55m - v5m
+
+    # making the shear vector
+    vl = [u6shr.magnitude, v6shr.magnitude, 0]
+
+    # Create a k hat vector
+    vk = [0, 0, 1]
+
+    # Take the cross product of the wind shear and k, and divide by the vector magnitude and
+    # multiply by the deviaton empirically calculated in Bunkers (2000) (7.5 m/s)
+    rdev = np.cross(vl, vk) * (7.5 / (np.sqrt(u6shr.magnitude ** 2 + v6shr.magnitude ** 2)))
+
+    # Add the deviations to the layer average wind to get the RM motion
+    urm = u6m.magnitude + rdev[0]
+    vrm = v6m.magnitude + rdev[1]
+
+    # Subtract the deviations to get the LM motion
+    ulm = u6m.magnitude - rdev[0]
+    vlm = v6m.magnitude - rdev[1]
+
+    rm_vector = np.asarray([urm, vrm]) * units('m/s').to(u.units)
+
+    lm_vector = np.asarray([ulm, vlm]) * units('m/s').to(u.units)
+
+    mean_vector = np.asarray([u6m.magnitude, v6m.magnitude]) * units('m/s').to(u.units)
+
+    return rm_vector, lm_vector, mean_vector
+
+
+@exporter.export
+@check_units('[pressure]', '[speed]', '[speed]')
+def bulk_shear(pressure, u, v, heights=None, bottom=None, depth=None):
+    r"""Calculate bulk shear through a layer.
+
+    Layer top and bottom specified in meters or pressure.
+
+    Parameters
+    ----------
+    pressure : `pint.Quantity`
+        Atmospheric pressure profile
+    u : `pint.Quantity`
+        U-component of wind.
+    v : `pint.Quantity`
+        V-component of wind.
+    height : `pint.Quantity`
+        Heights from sounding
+    depth: `pint.Quantity`
+        The depth of the layer in meters or hPa
+    bottom: `pint.Quantity`
+        The bottom of the layer in meters or hPa.
+        If in meters, must be in the same coordinates as the given
+        heights (i.e., don't use meters AGL unless given heights
+        are in meters AGL.) Default is the surface (1st observation.)
+
+    Returns
+    -------
+    `pint.Quantity'
```

Comment:
The returns here should be formatted like they are in Parameters:
```
type: name
   description
```

Can you format this and bunker's identically?